test/e2e/framework/framework.go

```diff
@@ -149,7 +149,9 @@ func (f *Framework) DeleteEtcdOperatorCompletely() error {
 	if err != nil {
 		return err
 	}
-	err = retryutil.Retry(2*time.Second, 5, func() (bool, error) {
+	// On k8s 1.6.1, grace period isn't accurate. It took ~10s for operator pod to completely disappear.
```

Comment:
can you create an issue in etcd operator repo as a tracker? we can move it to upstream when we have time. thanks.